Basic-Scripts/.idea/workspace.xml

```diff
@@ -0,0 +1,151 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Sneha can you please remove this workspace.xml file from the PR? This infact is an autogenerated file which is creating the merge conflict issue!

Reply:
Yes, I'm doing it right now

Reply:
> This code involves basic python syntax without the use of any external libarary.
> Please review it.

@snehagupta-4 nice and clean code.

Reply:
> > This code involves basic python syntax without the use of any external libarary.
> > Please review it.
> 
> @snehagupta-4 nice and clean code.

Thank you so much sir